veniq/dataset_collection/augmentation.py

```diff
@@ -75,6 +79,55 @@ def method_body_lines(method_node: ASTNode, file_path: Path) -> Tuple[int, int]:
     return start_line, end_line
 
 
+@typing.no_type_check
+def check_attrs(
```

Comment:
Those attrs are always exist for `ASTNode`. Is it this method where we want to distinguish from `None`? Then we should use `Optional` from typing.

Reply:
fixed